vars/unitTest.groovy

```diff
@@ -69,20 +74,50 @@ def call(Map config = [:]) {
 
   Map stage_info = parseStageInfo(config)
 
+  String inst_rpms = config['inst_rpms']
+
+  if (stage_info['compiler'] == 'covc') {
+    if (!inst_rpms) {
+      inst_rpms = ''
+    }
+    if (stage_info['target'].startsWith("centos")) {
+      inst_rpms += ' java-1.8.0-openjdk'
+    } else if (stage_info['target'].startsWith('ubuntu')) {
+      inst_rpms += ' openjdk-8-jdk'
+    } else {
+      // Assume a Suse distro
```

Comment:
Ditto for the assumption here, but perhaps this should be DRY'd out into a single function so that adding new distros only requires updating in one place.

Reply:
Yes, need to DRY it out.
Will fix in next push.